lib/src/godot_classes/dcl_global.rs

```diff
@@ -57,8 +57,7 @@ mod android {
             // "debug"  // Debug, info, warnings and errors (shows all debug logs)
             // "dclgodot::scene_runner=trace,warn"  // Trace for scene_runner, warn for everything else
             // "dclgodot::scene_runner=debug,dclgodot::comms=info,warn"  // Debug for scene_runner, info for comms, warn for everything else
-            "dclgodot::scene_runner::components::video_player=debug", // Debug for scene_runner, info for comms, warn for everything else
-                                                                      // "info", // Info, warnings and errors
+            "dclgodot::auth=debug", // Info, warnings and errors
```

Comment:
are we sure that we want this? i think `info` or `warn` should be the default... a lot of logs are going to be generated with this...